src/test/java/io/supertokens/test/ThirdPartyTest.java

```diff
@@ -454,14 +454,19 @@ public void testGetUsers() throws Exception {
 
         ThirdParty.signInUp(process.getProcess(), "thirdPartyId",
                 "thirdPartyUserId", "test@example.com", false);
+        Thread.sleep(100);
```

Comment:
Write a comment as to why you are pausing

Reply:
added comment